main.go

```diff
@@ -47,11 +48,11 @@ func initMageTools() {
 		}
 	}()
 
-	if mgtool.GetCWDPath("") != mgtool.GetGitRootPath("") {
+	if mgpath.FromCWD("") != mgpath.FromGitRoot("") {
 		panic(fmt.Errorf("can only be generated in git root directory"))
```

Comment:
Use `log.Panicf` or `log.Fatalf` instead of `panic(fmt.Errorf())`

Reply:
Im using panic because i have a defer func that hadles it, is that bad practice?
```go
	defer func() {
		if err := recover(); err != nil {
			logger.Error(err.(error), err.(error).Error())
		}
	}()
```

Reply:
Goit it, fixed